owncloudApp/src/main/res/values/strings.xml

```diff
@@ -137,7 +139,7 @@
     <string name="common_error">Error</string>
     <string name="common_loading">Loading&#8230;</string>
     <string name="common_unknown">unknown</string>
-    <string name="common_error_unknown">Unknown error</string>
+    <string name="common_error_unknown">unknown error</string>
```

Comment:
🤔

Reply:
I've changed the errors we are showing:

Instead of "Unknown error", we show "You can not create this due to unknown error" and I create these new errors dynamically. That's why we are using lower case now.